src/EditorFeatures/CSharpTest/RemoveUnusedMembers/RemoveUnusedMembersTests.cs

```diff
@@ -455,18 +455,19 @@ public async Task MethodIsUnused_Extern()
 class C
 {
     [DllImport(""Assembly.dll"")]
-    private static extern void [|M|]();
+    private static extern void M();
 }");
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedMembers)]
         public async Task MethodIsUnused_Abstract()
         {
-            await TestDiagnosticMissingAsync(
```

Comment:
Was this change required, given you still have this helper?

Reply:
➡️ Now covered by https://github.com/dotnet/roslyn/pull/41755#discussion_r381076580